rero_ils/modules/documents/views.py

```diff
@@ -434,3 +434,69 @@ def create_publication_statement(provision_activity):
     for publication_text in publication_texts:
         output.append(publication_text.get('value'))
     return output
+
+
+@blueprint.app_template_filter()
+def get_cover_art(record):
+    """Get cover art.
+
+    :param record: record
+    :return: url for cover art or None
+    """
+    for electronic_locator in record.get('electronicLocator'):
+        type = electronic_locator.get('type')
+        content = electronic_locator.get('content')
+        if type and type == 'resource' and content and content == 'coverImage':
+            return electronic_locator.get('url')
+    return None
+
+
+@blueprint.app_template_filter()
+def get_accesses(record):
+    """Get electronic locator text.
+
+    :param record: record
+    :return: dictonary list of access informations
+    """
+    accesses = []
+    electronic_locators = record.get('electronicLocator', [])
+    for electronic_locator in electronic_locators:
```

Comment:
Looks like a filter, you can probably use this: https://docs.python.org/3/library/functions.html#filter

Reply:
Implemented with filter.